pimpmycause/pimpmycause/templates/index.html

```diff
@@ -353,7 +353,7 @@ <h4 class="pnl__title">Sign-up to our newsletter</h4>
                                         <input type="email" name="email" placeholder="Email Address" required>
                                     </div>
                                     <div class="medium-6 large-6 small-12 columns">
-                                        <input class="button" type="submit" value="Singup">
+                                        <input class="button" type="submit" value="Sign-up">
```

Comment:
I had the same dilemma..  :) Do i leave the one below as it is though or change it also?
![image](https://cloud.githubusercontent.com/assets/13818956/18768412/e19a9ffe-811c-11e6-8dce-e226aca0494f.png)